examples/tutorial/jupyter/execution/pandas_on_dask/cluster/exercise_5.ipynb

```diff
@@ -0,0 +1,370 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![LOGO](../../../img/MODIN_ver2_hrz.png)\n",
+    "\n",
+    "<h1>Scale your pandas workflows by changing one line of code</h2>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Exercise 5: Setting up cluster environment\n",
+    "\n",
+    "**GOAL**: Learn how to set up a Dask cluster for Modin, connect Modin to a Dask cluster and run pandas queries on a cluster.\n",
+    "\n",
+    "**NOTE**: This exercise has extra requirements. Read instructions carefully before attempting. \n",
+    "\n",
+    "**This exercise instructs users on how to start a 500+ core Dask cluster, and it is not shut down until the end of exercise. Read instructions carefully.**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Often in practice we have a need to exceed the capabilities of a single machine. Modin works and performs well \n",
+    "in both local mode and in a cluster environment. The key advantage of Modin is that your python code does not \n",
+    "change between local development and cluster execution. Users are not required to think about how many workers \n",
+    "exist or how to distribute and partition their data; Modin handles all of this seamlessly and transparently.\n",
+    "\n",
+    "![Cluster](../../../img/modin_cluster.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Extra requirements for AWS authentication\n",
+    "\n",
+    "First of all, install the necessary dependencies in your environment:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install dask_cloudprovider[aws]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The next step is to setup your AWS credentials. One can set ``AWS_ACCESS_KEY_ID``, ``AWS_SECRET_ACCESS_KEY``\n",
```

Comment:
What about `aws configure`? Is there such an option?

Reply:
I think no, because ``aws configure`` requires interactive mode, but this is not possible for Jupyter Notebook.